lib/git.js

```diff
@@ -0,0 +1,13 @@
+var execSync = require("child_process").execSync;
+
+module.exports = {
+
+  branch: function(){
+    return execSync("git rev-parse --abbrev-ref HEAD || hg branch").toString().trim();
+  },
+
+  head: function(){
+    return execSync("git rev-parse HEAD || hg id -i --debug | tr -d '+'").toString().trim();
```

Comment:
@stevepeak This change seems to return the commit message instead of the commit ID.